jwst/straylight/straylight_step.py

```diff
@@ -3,45 +3,73 @@
 from ..stpipe import Step, cmdline
 from .. import datamodels
 from . import straylight
-
+from ..datamodels import RegionsModel
 
 class StraylightStep (Step):
     """
     StraylightStep: Performs straylight correction image using a Mask file.
     """
-    reference_file_types = ['straymask']
+
+    spec = """
```

Comment:
These new arguments also need to be added to the default `straylight.cfg` file in the pipeline directory.

Reply:
Do we put all the default values in the cfg files  ?
If so then I need to update the cube_build on too.

Reply:
Yes. See, for example, https://github.com/STScI-JWST/jwst/blob/master/jwst/pipeline/skymatch.cfg